### ISSUE TITLE
undefined method `each' for my single resource, because my model has a column "size".

### ISSUE THREAD

**Issue by AlbaHoo** [JSONAPI-Resources/jsonapi-resources#1323]
## This issue is a (choose one):

- [ ] Problem/bug report.
- [x] Feature request.
- [ ] Request for support. **Note: Please try to avoid submitting issues for support requests. Use [Gitter](https://gitter.im/cerebris/jsonapi-resources) instead.** 

## Checklist before submitting:

- [ ] I've searched for an existing issue.
- [ ] I've asked my question on [Gitter](https://gitter.im/cerebris/jsonapi-resources) and have not received a satisfactory answer.
- [x] I've included a complete [bug report template](https://github.com/cerebris/jsonapi-resources/blob/master/lib/bug_report_templates/rails_5_master.rb). This step helps us and allows us to see the bug without trying to reproduce the problem from your description. It helps you because you will frequently detect if it's a problem specific to your project.
- [ ] The feature I'm asking for is compliant with the [JSON:API](http://jsonapi.org/) spec.

## Description

Choose one section below and delete the other:

### Bug reports:
Here is the situation,  I have a model called "DeviceSecondaryModel" and it has a column name is "size", this is causing issues,

I have to dig into the source code and found this:
```
    def is_collection?(resource, force_is_collection = nil)
      return force_is_collection unless force_is_collection.nil?

      resource.respond_to?(:size) && !resource.respond_to?(:each_pair)
    end

```
So basically it treated my single resource as array, as it has a size method.  

I don't know this is a bug report or a feature request, just want to highlight here in case others have the same issue.

**Comment by AlbaHoo** [comment]
Sorry, wrong repo.
